app/src/main/java/ca/bc/gov/vaxcheck/di/DecoderModule.kt

```diff
@@ -84,4 +87,9 @@ class DecoderModule {
         context,
         shcConfig
     )
+
+    companion object {
+        private const val PROD_EXPIRY_TIME = 21600000L  // 6 Hours
+        private const val TEST_EXPIRY_TIME = 120000L    // 2 Minutes
```

Comment:
I'd suggest creating a value in the Constants file like:
```
const val SECONDS_IN_MILLIS = 1000
const val MINUTES_IN_MILLIS = 60 * SECONDS_IN_MILLIS
const val HOURS_IN_MILLIS = 60 * MINUTES_IN_MILLIS
```

then you'll be able to call:
```
6 * HOURS_IN_MILLIS
2 * MINUTES_IN_MILLIS
```

Reply:
on it

Reply:
Hmm I cannot stop thinking something... I understand that this change is to improve readability, however is it not better to put the final value so compiler doesn't have to do extra calculations?

Reply:
I mean, it doesn't matter anyways because const val are compile time constants which doesn't consume memory. but is not like those seconds_in_millis are going to be use elsewere

Reply:
The main idea is to improve readability. as you said, and make it easier to maintain, for instance: if you want to change to 7 hours, you'll need to recalculate, update code and update comment. 
Regarding the time to run: these calculations are not relevant on compile time when we compare to other tasks. Even if it was during runtime, calculations are not too time consuming for the type of app we are working on.
(same applies to mappings and other procedures we do because of architectural rules)